SonarQube.TeamBuild.PreProcessor/ProcessedArgs.cs

```diff
@@ -50,16 +51,21 @@ public ProcessedArgs(string key, string name, string version, bool installLoader
             {
                 throw new ArgumentNullException("globalFileProperties");
             }
+            if (scannerEnvProperties == null)
+            {
+                throw new ArgumentNullException("scannerEnvProperties");
```

Comment:
That's weird. The only thing that could make it fails is the version of the framework (which is fine).